frontend/src/metabase/querying/filters/components/FilterPicker/FilterColumnPicker/FilterColumnPicker.tsx

```diff
@@ -178,7 +174,7 @@ function getSections(
   ];
 }
 
-function renderItemName(item: ColumnListItem) {
+function renderItemName(item: ColumnListItem | SegmentListItem) {
```

Comment:
nit: `type Item = ColumnListItem | SegmentListItem` and we can reuse it 4 times in this file.